stable/mariadb/values.yaml

```diff
@@ -4,11 +4,11 @@
 ## Default: none
 image: bitnami/mariadb:10.1.19-r2
 
-## Specify a imagePullPolicy
-## Default to 'Always' if imageTag is 'latest', else set to 'IfNotPresent'
-## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
-##
-# imagePullPolicy:
+## Specify an imagePullPolicy (Required)
+## It's recommended to change this to 'Always' if the image tag is 'latest'
+## ref: http://kubernetes.io/docs/user-guide/images/#updating-images
+## Default: IfNotPresent
+imagePullPolicy: IfNotPresent
```

Comment:
Didn't see @prydonius comment. This may be contradictory. Either one or the other. I'm happy with either.